kanban-all/kanban-data/pom.xml

```diff
@@ -15,6 +15,11 @@
             <artifactId>spring-data-jpa</artifactId>
             <version>1.11.0.RELEASE</version>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>5.2.7.Final</version>
```

Comment:
also, please google for maven BOM model.

dependencies with their versions (as properties) are specified inside dependencyManagement  section of BOM (the root pom.xml)  as IMPORT dependencies.

submodules are just using these without version tag